jujugui/static/gui/src/app/components/store/store.js

```diff
@@ -22,34 +22,818 @@ YUI.add('store', function() {
+                              Rsyslog
+                            </h3>
+                            <p>by charmers</p>
+                        </div>
+                    </a>
+                </li>
+                <li className="three-col featured-entity">
+                    <a className="featured-entity__link"
+                      href="/?store=zabbix-server">
+                        <div
+                          className="one-col no-margin-bottom">
+                            <img
+                              src={this._generateIconPath('zabbix-server')}
+                              alt=""
+                              className="featured-entity__image control-size"/>
+                        </div>
+                        <div
+                          className="two-col last-col no-margin-bottom">
+                            <h3 className="featured-entity__title">
+                              Zabbix Server
+                            </h3>
+                            <p>by charmers</p>
+                        </div>
+                    </a>
+                </li>
+                <li className="three-col last-col featured-entity">
+                    <a className="featured-entity__link"
+                      href="/?store=~ricardokirkner/sentry/">
+                        <div
+                          className="one-col no-margin-bottom">
+                            <img src={this._generateIconPath(
+                              '~ricardokirkner/sentry')}
+                              alt=""
+                              className="featured-entity__image control-size"/>
+                        </div>
+                        <div
+                          className="two-col last-col no-margin-bottom">
+                            <h3 className="featured-entity__title">
+                              Sentry
+                            </h3>
+                            <p>by ricardokirkner</p>
+                        </div>
+                    </a>
+                </li>
+            </ul>
+            <p className="intro">
+                <a href="/?search=ops">
+                    View all operations&nbsp;&rsaquo;
+                </a>
+            </p>
+        </div>
+      </div>);
+    },
+
+    /**
+      The content for the bigdata section of the page
+
+      @method _bigDataSection
+      @return {Object} The contents of the section
+    */
+    _bigDataSection: function() {
+      return (<div className="row row--bigdata">
+        <div className="inner-wrapper clearfix">
+            <div className="six-col">
+                <h2>Big data charms and bundles</h2>
+                <p>Spend time testing, evaluating, and using your Big
+                Data solutions to benefit your business by using one of
+                Juju&rsquo;s pre-configured bundles or expertly
+                created charms.</p>
+                <p><a href="/?search=bigdata">
```

Comment:
You could link directly to charms/bundles that have the bigdata tag `/?search=&tags=bigdata` (though we may want the search, depends how good the results are I guess).